test/test.js

```diff
@@ -1,12 +1,33 @@
-import { expect } from 'chai';
-import request from 'request';
-
-describe('Test Server Response', () => {
-  describe('Server Response', () => {
-    it('status', () => {
-      request('http://localhost:8080/', (error, response) => {
-        expect(response.statusCode).to.equal(200);
-      });
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../server/app';
+
+import User from '../server/api/user/user.model';
+
+const should = chai.should();
```

Comment:
file: 'file:///Applications/MAMP/htdocs/meeting-for-good/test/test.js'
severity: 'Error'
message: ''should' is assigned a value but never used. (no-unused-vars)'
at: '7,7'
source: 'eslint'